src/views/ManageList.jsx

```diff
@@ -117,6 +68,7 @@ export function ManageList({ listPath, userId, data }) {
 					</label>
 					<button type="submit">Send invite!</button>
 				</form>
+				<button onClick={() => handleDelete()}>DELETE this list!</button>
```

Comment:
nit: I don't think we need a callback here, e.g.:

```suggestion
				<button onClick={handleDelete}>DELETE this list!</button>
```

Reply:
I wasn't sure about this, thanks for the correction! At first it seemed like the function was being triggered alongside rendering ManageList without it, so I put it in there to be safe.